content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -120,12 +120,11 @@ Once you have a Linux-based Kubernetes control-plane node you are ready to choos
 
     {{< note >}}
     If you're using a different interface rather than Ethernet (i.e. "Ethernet0 2") on the Windows nodes, you have to modify the line:
-    
+     {{< /note >}}
```

Comment:
@artisticcheese it's unusual to have a sentence that continues beyond the end of a note block. I agree that there is a bug but I'm not sure the fix you're suggesting is right formatting-wise either.

Reply:
Because ending `note` shall be after note itself while right now it includes `powershell` object as well

Reply:
Moving the code sample inside the note body should work.

This is definitely not the right fix.

Reply:
I don't understand but looks like preview looks correct?
https://deploy-preview-20132--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/